### ISSUE TITLE
formatting error when building with latest clang

### ISSUE THREAD

**Issue by matthargett** [JohnnyMorganz/luau-analyze-rojo#14]
I'm getting this warning, which may indicate an issue that would occur with file paths that include wide characters:
```
RequireResolver.cpp:187:49: warning: format specifies type 'wchar_t *' but the argument has type 'const std::filesystem::path::value_type *' (aka 'const char *') [-Wformat]
            printf("%*sPath: %ls\n", level, "", canonicalPath.c_str());
```

**Comment by JohnnyMorganz** [comment]
Fixed in https://github.com/JohnnyMorganz/luau-analyze-rojo/commit/9e8339102de025de9b22e3e9f3b405052e5f47b8
